widgetastic_manageiq/vm_reconfigure.py

```diff
@@ -30,31 +28,22 @@ class DisksTable(VanillaTable):
         "Mode": BootstrapSelect(id="hdMode"),
         "Size": Input(id="dvcSize"),
         "ControllerType": BootstrapSelect(id="Controller"),
-        # TODO: Workaround necessary until BZ 1524960 is resolved
-        5: BootstrapSelect(id="hdUnit"),  # for VMware in 5.9
-        4: BootstrapSelect(id="hdUnit"),  # for VMware in 5.8
-        3: BootstrapSelect(id="hdUnit"),  # for RHEVM in 5.9
-        "Dependent": VersionPick(
-            {
-                Version.lowest(): BootstrapSwitch(name="cb_dependent"),
-                "5.9": BootstrapSwitch(name="vm.cb_dependent"),
-            }
-        ),
+        "Unit": BootstrapSelect(id="hdUnit"),
+        "Dependent": BootstrapSwitch(name="vm.cb_dependent"),
         "Delete Backing": BootstrapSwitch(name="cb_deletebacking"),
         "Actions": Button(),
+        9: Button(),
```

Comment:
![table](https://user-images.githubusercontent.com/11618054/50308860-89963580-04c2-11e9-9f38-4acda4366177.png)

Attaching image provided by @niyazRedhat. This problem was faced by him. 
we have one extra column having `columnspan`. As per my understanding this is only best way to deal with this.

Reply:
I am not removed `Actions` attribute but just enhanced to fix those test. which was failing from few month. It will provide capability to FA owner to go for `resize` operation also.

Reply:
Can actions be a locally defined widget class with 2 buttons?

Reply:
@jawatts If I understand your idea is put view with action button having two button. Its not possible with current `TableColumn` implementation. If `header` not match we need to use column number.